### PR TITLE
update docs of kubeedge install prerequisites

### PR DESCRIPTION
**Pull request by kuramal**
Signed-off-by: Su Xiaolin <linxxnil@126.com>

/kind documentation

**What this PR does / why we need it**:
update docs of kubeedge install prerequisites, make it clearer

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -8,16 +8,23 @@ KubeEdge is composed  of cloud and edge parts. It is built upon Kubernetes and p
...
++ [Install docker](https://docs.docker.com/install/), or other runtime, such as [containerd](https://github.com/containerd/containerd)
 
-    you can also run other runtime, such as [containerd](https://github.com/containerd/containerd)
++ [Install kubeadm/kubectl](https://kubernetes.io/docs/setup/independent/install-kubeadm/)
 
-+ [Install kubeadm/kubectl on cloud side](https://kubernetes.io/docs/setup/independent/install-kubeadm/)
++ [Creating kubernetes cluster with kubeadm](<https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/>)
 
-+ [Creating kubernetes cluster with kubeadm on cloud side](<https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/>)
++ **Go** The minimum required go version is 1.12. You can install this version by using [this website.](https://golang.org/dl/)
```
Comment (fisherxu) [inline docs/setup/setup.md]: We can move L18 and L24 to L10 below `Prerequisites`.
```suggestion
+ **Go dependency** and **Kubernetes compatibility** please refer to [compatibility-matrix](https://github.com/kubeedge/kubeedge#compatibility-matrix).
```

**Comment by kuramal** [comment]
@fisherxu @kadisi PTAL

**Pull request merged**